components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -197,9 +194,7 @@ internal fun triggerWorkManager() {
         isWorkScheduled(PingUploadWorker.PING_WORKER_TAG))
 
     // Since WorkManager does not properly run in tests, simulate the work being done
-    GlobalScope.launch(Dispatchers.IO) {
-        PingUploadWorker.uploadPings()
-    }
+    PingUploadWorker.uploadPings()
```

Comment:
This just seems like good low-hanging fruit, but I'm not sure if there's a clear relationship to the failing tests.